### PR TITLE
075 player hand

### PR DESCRIPTION
**Pull request by SkyRiser568**
This feature adds a player hand object that can move around the screen.

Checklist before requesting a review:
- [X]  I have run this branch locally
- [X]  I have performed a self-review of my code
- [X]  I have confirmed critical features still function

### CODE REVIEW

File: MonstieWash/Assets/Scripts/PlayerHand.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        m_Transform = gameObject.transform;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        handPosition = Camera.main.WorldToScreenPoint(m_Transform.position);
+        moveHorizontal = Input.GetAxis("Horizontal");
+		moveVertical = Input.GetAxis("Vertical");
```
Comment (CrispyKorn) [inline MonstieWash/Assets/Scripts/PlayerHand.cs:29]: Just realizing this, but we'll have to pivot to the new input system at some point to allow for controller support. For now though, this is okay - just know that it's temporary. We can worry about that at a later date.

**Review by CrispyKorn — CHANGES_REQUESTED** [review]
This all looks great, there's just a couple things I'd like to see cleaned up before finalizing. The main one is (I'm not sure if it's just on my end but) there's some extra unnecessary indentation on some lines that make things look messy so it'd be great if you could line those up. (Also I wanna test and make sure I understand how requesting and implementing changes works!) :)

**Comment by emiko1412** [comment]
Oh i see you requested everyone in this one - i recommend only needing one programmer to do approval to not show things down too much. 

Dw about mistakes! We will be learning on how to get our processes down over time :3

File: MonstieWash/Assets/Scripts/PlayerHand.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    //Unity Inspector
+	[SerializeField]
+    [Range(1.0f, 50.0f)]
+	private float cursorSpeed = 20;
+
+    //Private
+    private Transform m_Transform;
+    private Vector3 handPosition;
+	private float moveHorizontal;
+	private float moveVertical;
```
Comment (CrispyKorn) [inline MonstieWash/Assets/Scripts/PlayerHand.cs:16]: I'm not sure if we've finalized this yet so don't worry about it too much for now, but it'd be good to be using the notation mentioned in the code guide on notion for all member variables (to specify that they are a member of this class). The `m_transform` is what you want to be doing for all member variables for now. (Like below)

```suggestion
    private Transform m_transform;
    private Vector3 m_handPosition;
	private float m_moveHorizontal;
	private float m_moveVertical;
```

**Review by CrispyKorn — CHANGES_REQUESTED** [review]
This all looks great, there's just a couple things I'd like to see changed before finalizing. Take a read through my comments and let me know if you've got any questions!

File: MonstieWash/Assets/Scripts/PlayerHand.cs
[the remark this answers is not in the document]
Comment (SkyRiser568) [reply MonstieWash/Assets/Scripts/PlayerHand.cs:1]: Yeah I see the strange indentations you mention. They don't show while I'm working inside vscode, not sure what happened, maybe something to do with indentation sizes between github and vscode, but I'll fix it up and keep an eye of for this in the future

File: MonstieWash/Assets/Scripts/PlayerHand.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (SkyRiser568) [reply MonstieWash/Assets/Scripts/PlayerHand.cs:29]: Yep. Just want to check that this is the right package/documentation
https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/index.html

[the remark this answers is not in the document]
Comment (CrispyKorn) [reply MonstieWash/Assets/Scripts/PlayerHand.cs]: Yep that's right, that's what we'll end up using I reckon.

File: MonstieWash/Assets/Scripts/PlayerHand.cs
[the remark this answers is not in the document]
Comment (CrispyKorn) [reply MonstieWash/Assets/Scripts/PlayerHand.cs:1]: Hmm that is weird, not sure why that would be happening. I'll double check when I submit my own code to see if it happens as I use VS too.

**Review by CrispyKorn — APPROVED** [review]
Looks good now, thanks for doing that! Feel free to merge into development and congrats on the first completed feature! 🎉

**Pull request merged**